open-metadata-implementation/user-interfaces/access-services-user-interface/src/main/java/org/odpi/openmetadata/userinterface/accessservices/server/DiscoveryServerInstanceHandler.java

```diff
@@ -0,0 +1,56 @@
+/* SPDX-License-Identifier: Apache 2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.discoveryserver.server;
```

Comment:
I was surprised to see the discovery server content in the UI - is it needed?

Reply:
@planetf1  I agree this does not look right. I think it might have crept in with the merge. I will take a look